### PR TITLE
Minor fixes and updates to CLI

### PR DESCRIPTION
**Pull request by ersonp**
Did you run `make format && make check`?
yes

Fixes #	

 Changes:	
- Changed Global variables to Local
- Minor updates in cmd flags
- Reverted struct `Common` to use the oath variable from it's own struct
- Removed unused param `args` from funcs
- Updated mac and windows path variables and PackageConfig
- Deleted unused code

How to test this PR:

### CODE REVIEW

File: cmd/skywire-cli/commands/config/gen.go [hunk 1]
```diff
@@ -121,24 +123,25 @@ var genConfigCmd = &cobra.Command{
 			logger.Fatal("Use of mutually exclusive flags: -w --hide and -n --stdout")
 		}
 		if dmsgHTTP {
+			dmsgHTTPPath := skyenv.DMSGHTTPPath
 			if pkgEnv {
```
Comment (ersonp) [inline cmd/skywire-cli/commands/config/gen.go:127]: @mrpalide @0pcom do we need to use a different dmsghttp config path for package?

File: pkg/visor/visorconfig/config.go [hunk 2]
```diff
@@ -111,13 +110,20 @@ func MakeBaseConfig(common *Common, testEnv bool, dmsgHTTP bool, services *Servi
...
 	if err != nil {
 		return nil, err
 	}
+
+	var dmsgHTTPServersList *DmsgHTTPServers
+
 	if dmsgHTTP {
-		serversListJSON, err := ioutil.ReadFile(skyenv.DMSGHTTPPath)
+		dmsgHTTPPath := skyenv.DMSGHTTPPath
+		if pkgEnv {
```
Comment (ersonp) [inline pkg/visor/visorconfig/config.go:122]: > @mrpalide @0pcom do we need to use a different dmsghttp config path for package?

same question here

File: cmd/skywire-cli/commands/config/gen.go [hunk 3]
```diff
@@ -67,7 +68,7 @@ func init() {
 	genConfigCmd.Flags().StringVarP(&hypervisorPKs, "hvpks", "j", "", "list of public keys to use as hypervisor")
 	genConfigCmd.Flags().StringVarP(&selectedOS, "os", "k", skyenv.OS, "(linux / macos / windows) paths")
 	genConfigCmd.Flags().BoolVarP(&stdout, "stdout", "n", false, "write config to stdout")
-	genConfigCmd.Flags().StringVarP(&output, "out", "o", "", "output config default:"+skyenv.ConfigName)
+	genConfigCmd.Flags().StringVarP(&output, "out", "o", skyenv.ConfigName, "output config default:"+skyenv.ConfigName)
```
Comment (0pcom) [inline cmd/skywire-cli/commands/config/gen.go]: this change messes up the formatting of the help menu. The way it was before was intentional for that reason

image

previously:

image

File: cmd/skywire-cli/commands/config/gen.go (hunk 1 above)
Comment (0pcom) [reply cmd/skywire-cli/commands/config/gen.go:127]: `/opt/skywire/dmsghttp-config.json` for the linux packages
something different for windows & mac I'm sure

**Comment by 0pcom** [comment]
image

wrong version

**Comment by 0pcom** [comment]
image
same text wrap-around issue for the visor help menu as with config gen on the cli

**Comment by ersonp** [comment]
> image
> 
> wrong version

I haven't changed anything about the version. You might need to pull the latest tags.
image

File: cmd/skywire-cli/commands/config/gen.go (hunk 3 above)
Comment (ersonp) [reply cmd/skywire-cli/commands/config/gen.go]: Fixed
image

**Comment by ersonp** [comment]
> image same text wrap-around issue for the visor help menu as with config gen on the cli

Fixed
image

File: cmd/skywire-cli/commands/config/gen.go (hunk 1 above)
Comment (mrpalide) [reply cmd/skywire-cli/commands/config/gen.go:127]: Main problem here is when trying to run `skywire-cli config gen ...` in a path different from location of files.
As Moses said for linux packages is `/opt/skywire/dmsghttp-config.json`.
For windows be good if we disable choosing install path by users, and just use `C:/Program Files/Skywire/dmsghttp-config.json`. Also same for Mac but I don't know about it.

Comment (ersonp) [reply cmd/skywire-cli/commands/config/gen.go:127]: I can add the windows and mac paths. Can someone just confirm the Mac path please? And this path `C:/Program Files/Skywire` would be the same for `skywire-config.json` too correct?

Comment (mrpalide) [reply cmd/skywire-cli/commands/config/gen.go:127]: Based on Mac Installer scripts, Mac path is correct.
And about the second question, we use another output for config generated, `"%HOMEPATH%\skywire-config.json"`.

Comment (ersonp) [reply cmd/skywire-cli/commands/config/gen.go:127]: So the `skywire-config.json` and `dmsghttp-config.json` are in a different location in the windows installer?

Comment (mrpalide) [reply cmd/skywire-cli/commands/config/gen.go:127]: Yes.

**Review by mrpalide — APPROVED** [review]
Approved for Linux and Windows.

**Comment by 0pcom** [comment]
approved for macos

**Review by 0pcom — APPROVED** [review]

**Pull request merged**